frontend/src/app/shared/enums/category-type.ts

```diff
@@ -1,7 +1,7 @@
 export enum CategoryType {
-    Fundamentals,
-    Algorithms,
-    BugFixes,
-    Refactoring,
-    Puzzles,
+    Fundamentals = 'Fundamentals',
+    Algorithms = 'Algorithms',
+    BugFixes = 'BugFixes',
```

Comment:
I think it would be better to add spaces whenever possible, so we can use it, for example, in our html templates